packages/graphqlsp/src/ast/token.ts

```diff
@@ -22,8 +22,8 @@ export const getToken = (
 
   let foundToken: Token | undefined = undefined;
   for (let line = 0; line < input.length; line++) {
-    const lPos = cPos - 1;
-    const stream = new CharacterStream(input[line]);
+    const lPos = cPos;
+    const stream = new CharacterStream(input[line] + '\n');
```

Comment:
Could it be that this line by itself is enough to fix https://github.com/0no-co/GraphQLSP/issues/190, so that we don't need the [regression](https://github.com/0no-co/GraphQLSP/issues/15) from the line before it?